### ISSUE TITLE
Items/Patterns Disappearing 

### ISSUE THREAD

**Issue by Darkosto** [raoulvdberge/refinedstorage#567]
#### Issue description:
Alright, so initially I thought this was due to switching versions but items and patterns are disapearing from my storage system every single client restart. I made the patterns in question in Beta 5, so I did not switch versions between losing these items. 

#### What happens:
My items are gone when loading the world, every single time. I notice it particularly happens with Solar Panel Controllers from Environmental Tech. Tier 1-4

#### What you expected to happen:  
Them to stay there

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: 2117
- Refined Storage: Beta 5

Does this issue occur on a server? [yes/no]

**Comment by raoulvdberge** [comment]
What do you mean with items/patterns dissapearing? Are items gone from your storage system? And are the actual pattern items gone from your Crafters? Also, did you try breaking and replacing your Controller?

**Comment by Darkosto** [comment]
I mean, they are deleted and do not exist anymore in my world.

**Comment by way2muchnoise** [comment]
I guess something goes wrong when they get reconstructed and it just removes them.

**Comment by Darkosto** [comment]
I also have a feeling, drives are being removed. I keep having to make new drives and when I restart my pack, I have less than when I left.

**Comment by raoulvdberge** [comment]
Any logs??

**Comment by raoulvdberge** [comment]
> I mean, they are deleted and do not exist anymore in my world.

I still have no idea what you mean, "deleted" is a very abstract term.

Which items are deleted? The items on your storage disks?

And how do patterns relate to this? Are they gone from your Crafters?

**Comment by Darkosto** [comment]
When I say deleted, I mean they are not in my world anymore. Patterns are gone from crafters. The items on disks are gone, but I believe the disks themsleves are actually gone, too. I don't have logs because I didn't catch this until it already happened.

**Comment by raoulvdberge** [comment]
Please try to reproduce this again and send me a log. I personally can't reproduce this, and I've been playing beta 5 on my private world since it came out.

**Comment by raoulvdberge** [comment]
This is most likely caused by a read fail and it should report *something* in your console.

**Comment by Darkosto** [comment]
1. Creation of patterns: https://sx.artdude.uk/hotlink-ok/darkosto/jMsc-80bd2-1478026312.mp4
2. Modpack exited and client restarted
3. Client restarted and missing patterns: https://sx.artdude.uk/hotlink-ok/darkosto/eMJb-1a829-1478026549.mp4

Logs: 
http://pastebin.com/rZmRb44P
https://www.dropbox.com/s/eg6gmx0qsclwafl/fml-client-latest.log?dl=0

**Comment by raoulvdberge** [comment]
1) Is your disk drive in another chunk than your controller?
2) Are your crafters in another chunk than your controller?
3) Can you still reproduce this after you *place a block* in the chunk of the crafters and disk drive and *then* reload?

**Comment by raoulvdberge** [comment]
4) Are the crafters and disk drives in the same chunk?

**Comment by raoulvdberge** [comment]
@Darkosto 
Please try this build:

https://drive.google.com/file/d/0B2PrXMobPWvWaE9mV010SnlLYzA/view?usp=sharing

**Comment by way2muchnoise** [comment]
Fixed as seen on stream \o/

**Comment by mrkarp** [comment]
Hello just an FYI, I don't know if its by design but we just experienced this. Turns out it was a solder being the 1st point of contact in the line of pipes, as in when items were traveling to the disk drives, the solder was before them and thus literally deleted the items that it did not take, or recognize for recipes instead of just letting them pass to the drives.

I can replicate if you would like and or give you any information you would need!

Thanks for this mod, just thought I'd see if this was supposed to happen.
